README.md

```diff
@@ -176,7 +176,7 @@ A curated list of awesome DevOps tools, technologies, platforms and resources. T
   - [Circle CI](https://circleci.com/) - powerful CI/CD pipelines that keep code moving.
   - [Bitrise](https://www.bitrise.io/) - CI/CD for mobile applications.
   - [Buildkite](https://buildkite.com/) - run fast, secure, and scalable continuous integration pipelines on your own infrastructure.
-  - [Cirrus CI](https://cirrus-ci.org/) - cycle fast, efficient, and secure with modern cloud technologies.
+  - [Cirrus CI](https://cirrus-ci.org/) - continuous integration system built for the era of cloud computing.
```

Comment:
From their [website](https://cirrus-ci.org/):
> Cirrus CI is a modern Continuous Integration system built for the era of cloud computing.

Thought this part of their description was a bit more descriptive of the product/service than:
> Make your development cycle fast, efficient, and secure